crates/paralegal-policy/src/context.rs

```diff
@@ -316,8 +330,10 @@ impl Context {
         to: &[&'a DataSink],
     ) -> Option<(&'a DataSource, &'a DataSink)> {
         from.iter().find_map(|&src| {
-            to.iter()
-                .find_map(|&sink| self.flows_to(ctrl_id, src, sink).then_some((src, sink)))
+            to.iter().find_map(|&sink| {
+                self.data_flows_to(ctrl_id, src, sink)
```

Comment:
If you're renaming `flows_to` to `data_flows_to`, then `any_flows` should probably be renamed to `any_data_flows_to`.

Or you can adopt the suggestion on my file comment instead

Reply:
Done